.github/workflows/drush-security-check.yml

```diff
@@ -0,0 +1,28 @@
+name: Drush Security Check
+on:
+  schedule:
+    - cron: '0 0 * * *'
+
+jobs:
+  check:
+    name: PHP ${{ matrix.php-versions }}
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        php-versions: ['8.1', '8.2']
```

Comment:
@vipin-mittal-acquia I think there's no need to check this on separate PHP version.

Reply:
@vishalkhode1 : Removed multiple PHP versions check and keep 8.1 default with ubuntu-latest